runtime/src/xcm_config.rs

```diff
@@ -138,16 +134,25 @@ pub type XcmRouter = WithUniqueTopic<(
     ChildParachainRouter<Runtime, XcmPallet, PriceForChildParachainDelivery>,
 )>;
 
+// zkVerify-EVM-Parachain
+pub const ZKV_EVM_PARA_ID: u32 = 1;
+// paratest, included in this repository
 pub const TEST_PARA_ID: u32 = 1599;
+
 parameter_types! {
-    pub const Acme: AssetFilter = Wild(AllOf { fun: WildFungible, id: AssetId(TokenLocation::get()) });
+    pub const TVFYAsset: AssetFilter = Wild(AllOf { fun: WildFungible, id: AssetId(TokenLocation::get()) });
```

Comment:
Let's simply call it tVFY ?

Reply:
Yeah we could do it (similar to Polkadot), but I don't quite like the fact that the name would overlap with our constants in the `currency` module.
Maybe we should also rename that, for example `VFY` -> `UNITS`, also similar to Polkadot's approach?

Reply:
I guess that's why polkadot named it using this case. As you wish, not a blocker, we can keep it like this